ember/app/components/project-form/component.js

```diff
@@ -58,6 +59,7 @@ export default class ProjectFormComponent extends Component {
       });
 
       this.router.transitionTo('projects.detailed', project.id);
+      created && project.unloadRecord();
```

Comment:
Use an if statement

Reply:
Could I haven an explanation as to why an if statement is better?

Reply:
Sure. This statement here indicates that `created` does something. However, you only use that because you want to unload the project **if** it was newly created. So using an `if` statement is what you actually want and is way better for readability as someone not familiar with the codebase would be wondering what the purpose of this statement is.

Reply:
Done here: https://github.com/adfinis/Outdated/pull/69/commits/8de0e14ff29d2a75e5da0b096c1596088fd69f7e

Reply:
:+1: If the reason for writing it like that was to keep it short, you can do `if (created) project.unloadRecord();`